cognee/modules/pipelines/operations/run_tasks_v2.py

```
@@ -0,0 +1,151 @@
+            additional_properties={
+                "pipeline_name": pipeline_name,
+            }
+            | config,
+        )
+
+        raise error
+
+
+async def run_tasks(
+    tasks: list[Task],
+    dataset_id: UUID = uuid4(),
```

Ruff (0.8.2)

Do not perform function call `uuid4` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)